examples/resources/credential/resource.tf

```diff
@@ -0,0 +1,140 @@
+# Create AWS_EKS credential
```

Comment:
Can you please add the resource_template files as well. And rename the PR as Acceptance test with examples..

Reply:
Resource_template file will be added during the documentation

Reply:
I mean we can do that way, but resource_templates are similar to examples, just that the values need to be replaced by placeholders. In documentation, we take up the templates and the generated docs, thoughts?

Reply:
The templating is better done during the Documentation task else going through the commit history might be confusing